docs/source/settingup.rst

```diff
@@ -18,17 +18,20 @@ The relevant requirements are listed below.
        If you don't have DataLad_ and its underlying tools (`git`_, `git-annex`_) installed yet, please follow the instructions from `the datalad handbook <http://handbook.datalad.org/en/latest/intro/installation.html>`_.
 
 
-Installation
-^^^^^^^^^^^^
+.. _feature_support:
+
+Feature support
+^^^^^^^^^^^^^^^^
+``datalad-dataverse`` is developed to be compatible with Dataverse (version 5.13), which
+has certain limitations when integrated with DataLad. In particular:
 
-.. attention:: **This extension is undergoing continous development and is in alpha stage!**
+- This extension does not support Dataverse versions prior to v5.13
+- This extension does not support unicode in filenames
+- Support for handling previously published Dataverse datasets is experimental
```

Comment:
maybe this last point isn't necessary as the package has never been released?

Reply:
I'm not sure what you mean. The intention of that point is to address https://github.com/datalad/datalad-dataverse/issues/241. Perhaps different wording would make it more obvious?